configs/components/windows_ruby.json

```diff
@@ -1,7 +1,7 @@
 {
   "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet-win32-ruby.git",
   "ref": {
-    "x86": "refs/tags/2.1.8.2-x86",
-    "x64": "refs/tags/2.1.8.2-x64"
+    "x86": "refs/tags/2.1.8.1-x86",
+    "x64": "refs/tags/2.1.8.1-x64"
```

Comment:
I think @Iristyle's comment was made before we decided to bump to openssl 1.0.2g in https://tickets.puppetlabs.com/browse/RE-6662, first released in puppet-agent 1.3.6 (and should be included in 1.4.0).